types/react-aria-menubutton/index.d.ts

```diff
@@ -80,7 +80,7 @@ export interface ButtonProps<T extends HTMLElement>
 export const Button: React.ForwardRefExoticComponent<ButtonProps<HTMLElement>>;
 
 export interface MenuProps<T extends HTMLElement>
-    extends React.HTMLProps<T> {
+    extends Omit<React.HTMLProps<T>, 'children'> {
```

Comment:
Could only extend `T`'s `children` because a function has an intersection with `{}`. Since we'll remove `{}` there won't be a valid intersection anymore. Maybe `ReactNode | ....` was intended?